docs/start/authentication/graphql-storefront.mdx

```diff
@@ -66,17 +66,227 @@ content-type: application/json
 </Tab>
 </Tabs>
 
+#### Customer access tokens
+
+A customer access token is unique to an individual user's account because it represents an authenticated storefront session for GraphQL requests. You can obtain and use a customer access token only for server-to-server requests. Therefore, you must use the customer access token with a regular storefront token. A customer access token becomes invalid on all devices when you log out of a single device. 
+
+<Callout type="warning">
+Do not use this token for browser-side or client-side requests. Browser-side requests require using the [customer impersonation token](#customer-impersonation-tokens).
```

Comment:
for browser-side requests `SHOP_TOKEN` cookie is used, customer impersonation token is also for server-to-server requests.